fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/NavigationBar.kt

```diff
@@ -0,0 +1,166 @@
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.theme.Theme
+
+/**
+ * Top-level UI for displaying the navigation bar.
+ *
+ * @param actionItems A list of [ActionItem] used to populate the bar.
+ */
+@Composable
+fun NavigationBar(actionItems: List<ActionItem>) {
+    Box(
+        modifier = Modifier
+            .background(FirefoxTheme.colors.layer1)
+            .height(48.dp)
+            .fillMaxWidth(),
+    ) {
+        Divider(color = FirefoxTheme.colors.borderPrimary)
+
+        Row(
+            modifier = Modifier
+                .align(Alignment.Center)
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
+            horizontalArrangement = Arrangement.SpaceBetween,
+        ) {
+            actionItems.forEach {
+                when (it.type) {
+                    ItemType.STANDARD -> {
+                        IconButton(onClick = {}) {
+                            Icon(
+                                painter = painterResource(it.iconId),
+                                stringResource(id = it.descriptionResourceId),
+                                tint = FirefoxTheme.colors.iconPrimary,
+                            )
+                        }
+                    }
+                    ItemType.TAB_COUNTER -> {
+                        IconButton(onClick = {}) {
+                            TabsIcon(item = it, tabsCount = 0)
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun TabsIcon(item: ActionItem, tabsCount: Int) {
+    Box(contentAlignment = Alignment.Center) {
+        Icon(
+            painter = painterResource(id = item.iconId),
+            contentDescription = stringResource(id = item.descriptionResourceId),
+            tint = FirefoxTheme.colors.iconPrimary,
+        )
+
+        Text(
+            text = tabsCount.toString(),
+            color = FirefoxTheme.colors.iconPrimary,
+            fontSize = 10.sp,
+            fontWeight = FontWeight.W700,
```

Comment:
Wondering if this patches with any of our typography?

Reply:
I am not sure I understand the question. What is out typography? 

The [component](https://www.figma.com/file/Kv7wWr8vH6AQrGhu9vmjdE/Mobile-Icons?type=design&node-id=221-1234&mode=design&t=5RfQ7uSnVewM1hFy-4) has the `FontWeight` of 700, and `TabsTrayBanner` uses the [same](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt#369) font weight for the element!

Reply:
I meant if this lined up with any of the defined font styles in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/theme/FenixTypography.kt#37